pkg/helper/taskrun/description/taskrun_description.go

```diff
@@ -23,6 +23,12 @@ const templ = `{{decorate "bold" "Name"}}:	{{ .TaskRun.Name }}
 {{decorate "bold" "Service Account"}}:	{{ .TaskRun.Spec.ServiceAccountName }}
 {{- end }}
 
+{{decorate "timeout" ""}} {{decorate "underline bold" "Timeout\n"}}
```

Comment:
Similar here as the comment for PipelineRun:

This should probably be similar to the following for Namespace:

```
{{decorate "bold" "Namespace"}}:	{{ .TaskRun.Namespace }}
```

So something like:

```
{{ if .TaskRun.Spec.Timeout not nil }} 
{{decorate "bold" "Timeout"}}:	{{ .TaskRun.Spec.Timeout.Duration.String }}
```

Reply:
Updated